packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionDetailTree.tsx

```diff
@@ -97,6 +101,26 @@ export const WfoSubscriptionDetailTree = ({
 
     if (!tree) return null;
 
+    /*
+     * The order of displayed product blocks should be the same as the order in the tree. Because we sort the tree
+     * alphabetically per level we can not depend on the order of product block ids anymore so we sort by the position in the tree.
+     */
+    const sortByTree = (idA: number, idB: number) => {
+        if (!tree) return 0;
+
+        const positionA = getPositionInTree(tree, idA);
+        const positionB = getPositionInTree(tree, idB);
+        if (!positionA || !positionB) return 0;
+
+        if (positionA < positionB) {
```

Comment:
Subtraction is not defined on the positionA type? (so you could do return positionA - positionB)

Reply:
Yes this could be a subtraction indeed

Reply:
The same answer as above ... I think